### DISCUSSION TITLE
Request to Get lastActivity return "Unauthorized"

### DISCUSSION THREAD

**Discussion by Valkovets** [eclipse-theia/theia-cloud#342] [Q&A] [answered]
http://10.0.34.132:3000/monitor/activity/lastActivity  => "error": "Unauthorized" What I just missed during configuration?

**Comment by jfaltermeier** [comment, marked as answer]
Hi, have you checked out the documentation for enabling monitoring at https://main--theia-cloud.netlify.app/documentation/addapplication/#enable-monitor-optional ?

To enable monitoring, you'll need to:
1. Add the `@eclipse-theiacloud/monitor-theia` npm package or use our VS Code extension, available 
 for each release, e.g. https://github.com/eclipsesource/theia-cloud/releases/download/0.11.1/theiacloud-monitor-0.11.1.vsix

2. Adjust the `monitor` port in your app definition:
   - Use port `8081` for the VS Code extension.
   - Use the same port as Theia if you're using the npm package, e.g. `3000`
